activerecord/lib/active_record/test_databases.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "active_support/testing/parallelization"
+
+module ActiveRecord
+  module TestDatabases
```

Comment:
I think we should express this module as private api by `# :nodoc:`

Reply:
I haven't decided yet if I want to make this private or document it. I think it could be useful for setting up test dbs when you have multiple databases.

Reply:
Decided to no doc this for now. Easier to document later than undocument.